### ISSUE TITLE
[Vertical tabs] Frame button is mispositioned on Linux

### ISSUE THREAD

**Issue by sangwoo108** [brave/brave-browser#33928]
When hiding window title with vertical tabs, frame button is laid out on wrong position

![image](https://github.com/brave/brave-browser/assets/5474642/27c295d1-0959-48eb-8777-084da2fb4b37)

**Comment by kjozwiak** [comment]
So I can still reproduce the above on my `Pop OS 22.04 x64` VM but both @btlechowski & @rillian confirmed that it's working on their VM/machines as per. We'll uplift/release into `1.60.x` and will create a follow up if we get anymore reports of the above still not working for certain users (not sure if it's my VM or still an issue on certain distro's).

Example of the issue still happening on my `PopOS 22.04 x64` VM:

```
Brave | 1.62.34 Chromium: 119.0.6045.105 (Official Build) nightly (64-bit)
--- | ---
Revision | 52fce576add8baed99146214aa452f5c75462738
OS | Linux
```

![regressionLinux](https://github.com/brave/brave-browser/assets/2602313/6193a597-6e1f-46e0-ad64-1d6697cb0373)

**Comment by kjozwiak** [comment]
The above requires `1.60.114` or higher for `1.60.x` verification 👍

**Comment by btlechowski** [comment]
Verified with

Brave | 1.60.114 Chromium: 119.0.6045.124 (Official Build) (64-bit)
-- | --
Revision | 99a34933fc6a44fdfe52dea41341927bbce3549a
OS | Ubuntu 18.04 LTS

Verified Frame button is positioned properly

![image](https://github.com/brave/brave-browser/assets/34715963/4168260a-d2be-4110-8e63-1f977f48f21a)|![image](https://github.com/brave/brave-browser/assets/34715963/bb8469d3-73c4-41ec-bbef-2156751f61c0)|![image](https://github.com/brave/brave-browser/assets/34715963/3f6d538f-c81a-4438-94fe-0ed1a68467e2)|![image](https://github.com/brave/brave-browser/assets/34715963/3404da88-1e86-4eb1-b626-fa4f983c3001)
--|--|--|--

----------------------------

Verified with

Brave | 1.60.114 Chromium: 119.0.6045.124 (Official Build) (64-bit)
-- | --
Revision | 99a34933fc6a44fdfe52dea41341927bbce3549a
OS | Ubuntu 23.04 LTS

Verified Frame button is positioned properly

![image](https://github.com/brave/brave-browser/assets/34715963/821d6f31-abe4-4f51-91a8-c743ce17e688)|![image](https://github.com/brave/brave-browser/assets/34715963/a6dadaf3-3890-4726-a4e7-125dd0fe09fd)|![image](https://github.com/brave/brave-browser/assets/34715963/58208de5-0f57-4dcf-bb23-e63e175f0b7a)
--|--|--

**Comment by rillian** [comment]
Confirming fixed in Brave 1.61.83 Chromium: 119.0.6045.163 (Official Build) beta (64-bit), rpm package on gnome-shell.
